### PR TITLE
bug 957802 - serve fonts to local.allizom for devs

### PR DESCRIPTION
**Pull request by groovecoder**
spot-check: `vagrant provision` and then check a page that tries to load `compat-icons.woff`: the request should succeed instead of a 403.

### CODE REVIEW

File: provisioning/roles/kuma/files/apache2/all-servers.conf [hunk 1]
```diff
@@ -11,11 +11,13 @@ Header set Access-Control-Allow-Credentials "false"
     # Only enable CORS for fonts when the request is coming from a Mozilla domain
     Header unset Access-Control-Allow-Origin
     SetEnvIf Origin "https?://(.*\.mozilla\.(com|org|net))" CORS=$0
```
Comment (darkwing) [inline provisioning/roles/kuma/files/apache2/all-servers.conf]: SetEnvIf Origin "https?://(.*\.(mozilla|allizom)\.(com|org|net))" CORS=$0

File: provisioning/roles/kuma/files/apache2/all-servers.conf [hunk 2]
```diff
@@ -11,11 +11,13 @@ Header set Access-Control-Allow-Credentials "false"
...
+    SetEnvIf Origin "https?://(.*\.allizom\.(com|org|net))" CORS=$0
     SetEnvIf Origin "https?://(mdn\.mozillademos\.org)" CORS=$0
     Header set Access-Control-Allow-Origin %{CORS}e env=CORS
 
     # block hotlinking by referer to .woff, .eof, .ttf files except mozilla domains
     RewriteCond "%{HTTP_REFERER}" "!https?://.*\.mozilla\.(com|org|net)/.*$"
+    RewriteCond "%{HTTP_REFERER}" "!https?://.*\.allizom\.(com|org|net)/.*$"
```
Comment (darkwing) [inline provisioning/roles/kuma/files/apache2/all-servers.conf]: (same as above)

**Comment by stephaniehobson** [comment]
Watch this when it goes from staging to production, we can't test that it still works in prod, only dev.

**Comment by stephaniehobson** [comment]
R+

File: provisioning/roles/kuma/files/vagrant/settings_local.py [hunk 3]
```diff
@@ -32,7 +32,7 @@
 
 INSTALLED_APPS = INSTALLED_APPS + (
     "django_extensions",
-    "debug_toolbar",
+#    "debug_toolbar",
```
Comment (darkwing) [inline provisioning/roles/kuma/files/vagrant/settings_local.py]: Did you mean for this to be in here, @groovecoder ?

Comment (groovecoder) [inline provisioning/roles/kuma/files/vagrant/settings_local.py]: Actually yes - it breaks our local selenium tests and @stephaniehobson noted it's a general pain for front-end after every provision. Could have been a separate commit I suppose, which I'm happy to do.

**Comment by darkwing** [comment]
R+, money!

**Pull request merged**